src/test/java/com/hierynomus/msdfsc/DFSTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C)2016 - SMBJ Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hierynomus.msdfsc;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.bouncycastle.util.encoders.Hex;
+import org.junit.Test;
+
+import com.hierynomus.msdfsc.DFSReferral;
+import com.hierynomus.msdfsc.SMB2GetDFSReferralResponse;
+import com.hierynomus.protocol.commons.buffer.Buffer.BufferException;
+import com.hierynomus.smbj.SMBClient;
+import com.hierynomus.smbj.auth.AuthenticationContext;
+import com.hierynomus.smbj.common.SMBBuffer;
+import com.hierynomus.smbj.connection.Connection;
+import com.hierynomus.smbj.session.Session;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class DFSTest{
```

Comment:
It's already done, in SMB2GetDFSReferralResponseTest.groovy and DFSTest.groovy.  I guess I'll delete this Java file.